migration-engine/connectors/sql-migration-connector/src/sql_renderer/mysql_renderer.rs

```diff
@@ -192,10 +198,13 @@ impl SqlRenderer for MysqlFlavour {
         };
 
         match foreign_key {
-            Some(_) => format!("{} {} {} {}", column_name, tpe_str, nullability_str, default_str),
+            Some(_) => format!(
+                "{}{} {}{}{}",
```

Comment:
you might wanna name these here, it's kind of hard to read...

```rust
"{indent}{column_name}..."
```

Reply:
definitely — I am planning to move all the string manipulation to structs with `Display` impls, so this won't stay forever, but we can definitely improve on this